src/utils/update/company.profile.mapper.to.oversea.entity.ts

```diff
@@ -0,0 +1,35 @@
+import { ICompanyDetails } from "model/company.profile.model";
+import { Address } from "../../model/data.types.model";
+
+export const mapCompanyProfileToOverseasEntityToDTOUpdate = (data: any): ICompanyDetails => {
+  if (!data){
+    return {} as ICompanyDetails;
+  }
+  return {
+    companyName: data?.companyName,
+    dateOfCreation: data?.dateOfCreation,
+    companyType: data?.type,
+    companyNumber: data?.companyNumber,
+    registeredOfficeAddress: mapCompanyAddressToOverseasEntityAddressDTO(data?.registeredOfficeAddress), // principal address
+    // incorporationCountry: data?.foreign_company_details?.originating_registry?.country,
+    // isOnRegisterInCountryFormedIn: mapYesOrNo(data),
+    // serviceAddress: mapCompanyAddressToOverseasEntityAddressDTO(data?.registeredOfficeAddress),
+    // legalForm: data?.legal_form,
+    // lawGoverned: data?.foreign_company_details?.governed_by,
+    // publicRegisterJurisdiction: data?.jurisdiction,
+    // registrationNumber: data?.registration_number,
+    // isServiceAddressSameAsPrincipalAddress: mapYesOrNo(data)
+  };
+};
+
+
+export const mapCompanyAddressToOverseasEntityAddressDTO = (registeredOfficeAddress: any): Address => {
+  return {
+    line_1: registeredOfficeAddress?.addressLineOne,
+    line_2: registeredOfficeAddress?.addressLineTwo,
+    town: registeredOfficeAddress?.locality,
+    county: registeredOfficeAddress?.region,
+    country: registeredOfficeAddress?.country,
+    postcode: registeredOfficeAddress?.postalCode
```

Comment:
include the field property_name_number

property_name_number: cp.serviceAddress?.premises,
line_1: cp.serviceAddress?.addressLineOne,
line_2: cp.serviceAddress?.addressLineTwo,
town: cp.serviceAddress?.locality,
county: cp.serviceAddress?.region,
country: cp.serviceAddress?.country,
postcode: cp.serviceAddress?.postalCode,